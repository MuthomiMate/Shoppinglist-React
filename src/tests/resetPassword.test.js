import React from 'react'
import PassReset from '../resetPassword'
import ReactDOM from 'react-dom'
import Enzyme from 'enzyme'
import {shallow,mount} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter:new Adapter()})
describe ('<PassReset/>', () =>{
    it('pass reset renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<PassReset/>, div);
    });
    it('changes email state', () => {
        const wrapper = mount(<PassReset />);
        const input = wrapper.find('#email');
        const target = {
            value: 'muthomi@gmail.com',
        };
        input.simulate('change', { target });
        expect(wrapper.state().email).toEqual(target.value);
    })
})


