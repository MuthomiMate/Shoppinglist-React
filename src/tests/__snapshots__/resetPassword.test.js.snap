// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PassReset/> renders page without crashing  1`] = `
<div>
  <div />
  <div
    className=" col-lg-offset-4 col-md-4 "
  >
    <div
      className="panel panel-success"
    >
      <div
        className="panel-heading"
      >
        Password Reset
      </div>
      <div
        className="panel-body"
      >
        <form
          onSubmit={[Function]}
        >
          <div
            className="form-group"
          >
            <input
              className="form-control"
              id="email"
              onChange={[Function]}
              placeholder="Enter your Email"
              required={true}
              type="email"
            />
          </div>
          <div
            className="form-group"
          >
            <input
              className="btn btn-primary"
              type="submit"
              value="submit"
            />
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;
